premake5.lua

```diff
@@ -337,6 +338,15 @@ if VST24SDK then
 
 	if (os.istarget("macosx")) then
 
+		if XCODE9 then
+			xcodebuildsettings
+			{
+				["CC"] = "/usr/local/opt/llvm/bin/clang";
```

Comment:
I'm not sure any of this sort of build stuff is necessary anymore. I've [implemented enough of the filesystem API](https://github.com/kurasu/surge/blob/master/libs/filesystem/filesystem.cpp) that you can just use the normally included clang with XCode 9+.